src/applications/personalization/profile/hooks/useNotificationSettingsUtils.js

```diff
@@ -192,17 +244,38 @@ export const useNotificationSettingsUtils = () => {
   const useFilteredItemsForMHVNotifications = itemIds =>
     useMemo(
       () => {
-        return toggles.showMhvNotificationSettings
-          ? itemIds
-          : itemIds.filter(itemId => {
-              return !BLOCKED_MHV_NOTIFICATION_IDS.includes(itemId);
-            });
+        // Always exclude these items: ticket# 89524
+        const alwaysExcludedItems = ['item7', 'item8', 'item11', 'item12'];
+
+        return itemIds.filter(itemId => {
+          // always exclude items in alwaysExcludedItems
+          if (alwaysExcludedItems.includes(itemId)) {
+            return false;
+          }
+          // exclude item9 if profileShowMhvNotificationSettingsNewSecureMessaging is turned off
+          if (
+            itemId === 'item9' &&
```

Comment:
Similar to the previous comment, this could be refactored to not use a magic string identifier